husky_components/column-navigation/main.js

```diff
@@ -1344,7 +1344,9 @@ define(function() {
          */
         mark: function(id, item) {
             var $element = this.$find('li[data-id="' + id + '"]');
-            if (!!$element.length) {
+
+            // mark only non-ghost pages if showActionButtonOnGhost is disabled
+            if (!!$element.length && ((!!item.type || item.type.name !== 'ghost') || this.options.showActionButtonOnGhost)) {
```

Comment:
Still getting the `Cannot read property 'name' of undefined` error. Probably because it should be `!!item.type && item.type.name !== 'ghost'` instead of `||`. Not sure if that covers the entire problem though.